CHANGELOG.md

```diff
@@ -19,6 +19,8 @@
 
 ### Fixed
 - virtio-blk: VIRTIO_BLK_T_FLUSH now working as expected.
+- The vsock feature can be used when starting Firecracker using the jailer
```

Comment:
I would split this in 2:
- in the added section: Jailer support for firecracker vsock builds.
- in the fixed section: Updated seccomp whitelist to allow seccomp level = 2 with vsock